tests/plugins/colordialog/manual/showcurrentcolor.md

```diff
@@ -0,0 +1,29 @@
+@bender-tags: feature, 4.12.0, 2639
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, colorbutton, colordialog
+
+**Scenario 1:**
```

Comment:
This scenario is not true, as in such case no color is selected and the first color in the dialog is focused.